test/end-to-end/shared/TestUtils.js

```diff
@@ -370,7 +370,7 @@ module.exports = {
     await this.input(model, label, anchor);
 
     // select the item of the dropdown menu matching the label
-    return anchor.locator(`.dropdown-menu > [role="option"] >> text="${label}"`).click();
+    await anchor.locator(`.dropdown-menu > [role="option"] >> text="${label}"`).click();
```

Comment:
This  function is advertised as returning a promise.  It used to do that with the final 'return...'.   Now it does not do that.  Was there a reason for this change?

Reply:
Nope, fixed.